### ISSUE TITLE
Bug with one-liner object literals.

### ISSUE THREAD

**Issue by Bartvds** [palantir/tslint#11]
This oneliner literal:

```
var valueA = {a: 1, b: 'B', c: [1, 2, 3], d: {a: 11, b: 'bravo', c: [11, 22, 33]}};
```

First gives this warning:

```
>> test/modules/xm/hash.ts[39, 73]: duplicate key 'a'
>> test/modules/xm/hash.ts[39, 79]: duplicate key 'b'
```

Later followed by:

```
>> test/modules/xm/hash.ts[39, 18]: unexpected tab width: expected 16, got 0
>> test/modules/xm/hash.ts[39, 32]: unexpected tab width: expected 16, got 0
>> test/modules/xm/hash.ts[39, 36]: unexpected tab width: expected 20, got 0
>> test/modules/xm/hash.ts[39, 48]: unexpected tab width: expected 20, got 0
>> test/modules/xm/hash.ts[39, 55]: unexpected tab width: expected 20, got 0
>> test/modules/xm/hash.ts[39, 73]: unexpected tab width: expected 16, got 0
>> test/modules/xm/hash.ts[39, 79]: unexpected tab width: expected 16, got 0
```
